strawberryfields/api/connection.py

```diff
@@ -135,6 +136,29 @@ def use_ssl(self) -> bool:
         """
         return self._use_ssl
 
+    def get_device(self, target: str) -> DeviceSpec:
```

Comment:
(Minor suggestion)
```suggestion
    def get_device_spec(self, target: str) -> DeviceSpec:
```

Reply:
I suppose these two are synonymous in a way?

Reply:
Hmm, generally speaking, I think `get_device` would imply to the reader that by calling this method we obtain some sort of device object. In the context of SF this could be associated as an engine, but for example, it would fit the terminology of PennyLane well as there are `Device` objects there.

As a specific example in SF, a `RemoteEngine` instance stores a `Connection` object that can be accessed with `RemoteEngine.connection`. One could then place a call like `RemoteEngine.connection.get_device` which in itself does not entirely describe that we get details about the device that is being run.

My point is that the reader has to have the background knowledge here that `get_device` returns the specification of the device which I believe could be conveyed by renaming the method.

Reply:
You've convinced me, I've changed it :)